src/JsonApi/Serializer/ItemNormalizer.php

```diff
@@ -323,14 +345,69 @@ private function getPopulatedRelations($object, string $format = null, array $co
                 if (!isset($attributeValueElement['data'])) {
                     throw new InvalidArgumentException(sprintf('The JSON API attribute \'%s\' must contain a "data" key.', $relationshipName));
                 }
-
+                unset($attributeValueElement['data']['attributes']);
                 $data[$relationshipName]['data'][] = $attributeValueElement['data'];
             }
         }
 
         return $data;
     }
 
+    /**
+     * Populates relationships keys.
+     *
+     * @param object      $object
```

Comment:
All the `param` annotations and the `return` one can be removed (useless, already documented by the typehints)

Reply:
done